shell.nix

```diff
@@ -1,18 +1,46 @@
-with import ((import ./lib.nix).fetchNixPkgs) { };
+{ nixpkgs ? import <nixpkgs> {}, compiler ? "default", doBenchmark ? false }:
 
 let
-  hsPkgs = haskell.packages.ghc822;
+
+  inherit (nixpkgs) pkgs;
+
+  f = { mkDerivation, aeson, array, attoparsec, base, bytestring
+      , containers, directory, generics-sop, hspec, http-conduit
+      , monad-control, mtl, optparse-applicative, QuickCheck, reflection
+      , regex-tdfa, resource-pool, sqlite-simple, stdenv, text, time
+      , transformers-base, universum, zip-archive
+      }:
+      mkDerivation {
+        pname = "log-classifier";
+        version = "1.1.0.0";
```

Comment:
Hm. This seems to be duplicated. Can't you import this since we have it already?